krod/main.py

```
@@ -160,7 +162,7 @@ def main() -> None:
         
         if args.query:
             # Process single query mode
-            engine = KrodEngine(config)
+            engine = AgentEngine(config)
```

Pylint (3.3.7)

[error] No value for argument 'memory_manager' in constructor call

(E1120)